### PR TITLE
🐛 Fixed Redis issue where keys ended up in the wrong deployment.

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

After the upgrade of python `3.11` enums work a bit differently. `class X(int, Enum)` and `class X(IntEnum)` work differently when serialising them in a string. To maintain the same behaviour it is required to use the `IntEnum`

Previous serialisation `redis://:***@redis:6379/RedisDatabase.ANNOUNCEMENTS` -> current fixed one `redis://:***@redis:6379/5`

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by giancarloromeo — APPROVED** [review]

File: packages/settings-library/src/settings_library/redis.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from .basic_types import PortInt
 
 
-class RedisDatabase(int, Enum):
+class RedisDatabase(IntEnum):
```
Comment (sanderegg) [inline packages/settings-library/src/settings_library/redis.py:11]: Anywhere else we might have the same issue?

**Review by sanderegg — APPROVED** [review]
Did you check if there are other occurences of this?

File: packages/settings-library/src/settings_library/redis.py (hunk 1 above)
Comment (GitHK) [reply packages/settings-library/src/settings_library/redis.py:11]: Nope, I've checked. this was the only one left

**Review by matusdrobuliak66 — APPROVED** [review]
super!

**Review by pcrespov — APPROVED** [review]
thx! :tada:

**Pull request merged**